build-linux-client.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
```

Comment:
What should RELEASE be when using this / mention this in the readme / developer doc